blog/2023-06-06-tolgee-translator.md

```diff
@@ -9,7 +9,7 @@ tags: [translation, ai, chatgpt, in-context, sdk, i18n]
 
 Do you think that AI will replace human translators? AI has made its way into various aspects of our lives, including localization, where it demonstrates exceptional capabilities. Yet, two significant challenges hinder the complete adoption of AI for software translation: reliability and context. Tolgee AI Translator aims to tackle these challenges head-on to provide contextually tailored translations of superior quality.
 
-![Tolgee translator](/img/blog/tolgee-translator/post-ai.png)
+![Tolgee AI translator](/img/blog/tolgee-translator/post-ai.png)
```

Comment:
Have we agreed on this? Personally I would not flex it so much. All the translators use some kind of AI, so one may ask "What about it?"

Reply:
@stepan662

Reply:
@JanCizmar I was just making the naming consistent.

Reply:
I also feel that everything is AI these days, it's kinda cheap label